client/src/components/ElementEditor/ElementList.js

```diff
@@ -29,10 +31,14 @@ class ElementList extends Component {
   renderBlocks() {
     const { ElementComponent, blocks } = this.props;
 
+    // Blocks can be either null or an empty array
```

Comment:
If blocks is null do you not want to show the message below?

Reply:
No. Blocks will be null/undefined while the GraphQL query is running I think. It will be an empty array if there's nothing

Reply:
Yes, what @ScopeyNZ is saying is correct. I should have made that clearer. By returning nothing on blocks being null we prevent that the message _and_ the loading component are rendered at the same time, which looks really ugly.

Reply:
Perhaps reflecting this information in the comment would be good :)